package.json

```diff
@@ -27,5 +30,16 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "cypress": "^13.13.1",
+    "cypress-plugin-api": "^2.11.1",
+    "start-server-and-test": "^2.0.4"
+  },
+  "dependencies": {
```

Comment:
There should be no need for these dependencies.

Can you explain why you added them, please?

Reply:
I agree, they've been removed!